apps/channels/tests/message_examples/test_telegram_channel.py

```diff
@@ -86,3 +111,45 @@ def test_handle_telegram_api_error_other_errors():
 
     assert expected_channel_exception_message == str(excinfo.value)
     assert excinfo.value.__cause__ == api_error
+
+
+def test_supported_image_file(telegram_channel, make_mock_file):
+    image_file = make_mock_file("img.jpg", "image/jpeg", 5 * 1024 * 1024)
```

Comment:
My apologies, I didn't see that the `make_mock` is parameterized. In this case, it makes more sense to do what you initially did where you called a function directly. Using a fixture doesn't quite make sense here. My apologies